uk.co.compucorp.civicrm.hrleaveandabsences/js/angular/src/leave-absences/shared/controllers/calendar-ctrl.js

```diff
@@ -248,7 +248,9 @@ define([
     this._loadAbsenceTypes = function () {
       return AbsenceType.all()
         .then(function (absenceTypes) {
-          this.absenceTypes = absenceTypes;
+          this.absenceTypes = absenceTypes.filter(function (absenceType) {
+            return absenceType.is_active === "1";
```

Comment:
```js
return absenceType.is_active;
```
should be enough, if you use `===` you might encounter problem in case the api returns 1 as an int